src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -124,15 +170,28 @@ export class PointerTool extends BaseTool {
     }
 
     sceneController.hoveredGlyph = undefined;
+    const resizeHandle = this.getResizeHandle(initialEvent, initialSelection);
 
-    if (initiateRectSelect) {
+    if (initiateRectSelect && !resizeHandle) {
       return await this.handleRectSelect(eventStream, initialEvent, initialSelection);
-    } else if (initiateDrag) {
+    } else if (initiateDrag && !resizeHandle) {
       this.sceneController.sceneModel.initialClickedPointIndex =
         initialClickedPointIndex;
       const result = await this.handleDragSelection(eventStream, initialEvent);
       delete this.sceneController.sceneModel.initialClickedPointIndex;
       return result;
+    } else if (
+      resizeHandle &&
+      this.editor.visualizationLayersSettings.model["fontra.resize.selection"]
```

Comment:
Please do this check earlier, at `const resizeHandle`:
```js
    const resizeHandle = this.editor.visualizationLayersSettings.model["fontra.resize.selection"]
      ? this.getResizeHandle(initialEvent, initialSelection)
      : undefined;
```
Or (maybe even better?), add the check to getResizeHandle().

Reply:
Thanks. I moved it to getResizeHandle.